### PR TITLE
Exclude sbom binaries

### PR DESCRIPTION
**Pull request by johnoliver**
# Checklist

- [ ] You added tests to cover the change
- [ ] `mvn clean install` build and test completes
- [ ] You changed or added to the documentation

### CODE REVIEW

**Comment by smlambert** [comment]
We will eventually want to serve the sbom files up from the API.  Having said that, they will not be .tar.gz, but the .json file directly (refactored via https://github.com/adoptium/temurin-build/pull/2955).

**Comment by johnoliver** [comment]
@smlambert unfortunately in the short term the sbom files being uploaded incorrectly label themselves as jdk images in their metadata, which is causing download issues, so need to be excluded for now until that can be fixed

**Comment by smlambert** [comment]
Yep, understand.  Just wanted to let you know of the intent, and now I also see this related: https://github.com/adoptium/api.adoptium.net/pull/441

File: adoptium-updater-parent/adoptium-mappers-parent/adopt-mappers/src/main/kotlin/net/adoptium/api/v3/mapping/adopt/AdoptReleaseMapper.kt [hunk 1]
```diff
@@ -24,15 +24,15 @@ import net.adoptium.api.v3.parser.VersionParser
 import org.slf4j.LoggerFactory
 import java.security.MessageDigest
 import java.time.ZonedDateTime
-import java.util.Base64
+import java.util.*
```
Comment (karianna) [inline adoptium-updater-parent/adoptium-mappers-parent/adopt-mappers/src/main/kotlin/net/adoptium/api/v3/mapping/adopt/AdoptReleaseMapper.kt:27]: Can we go back to the explicit import?

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**